create_graphics.py

```diff
@@ -490,9 +495,9 @@ def graphics_driver(cla):
                 create_maps(cla, fhr, grib_path, workdir)
 
             # Zip png files and remove the originals in a subprocess
-            if zipf:
-                png_files = glob.glob(os.path.join(workdir, f'*{fhr:02d}.png'))
-
+            for tile, zipf in zipfiles.items():
+                png_files = glob.glob(os.path.join(workdir, f'*_{tile}_*{fhr:02d}.png'))
+                png_files = [png_file.replace("__", "_") for png_file in png_files]
```

Comment:
I'm a little unclear on what this replace is doing. How did we get two underscores in the filename?

Reply:
It was leftover from testing.  The double underscores are taken care of earlier.  That line has been removed.